src/CoreGraphics/CGColor.cs

```diff
@@ -131,7 +130,18 @@ public CGColor (string name)
 				CGColorRetain (handle);
 			}
 		}
-#endif
+
+		[iOS (14,0)][TV (14,0)][Watch (7,0)]
+		public CGColor (CGConstantColor color)
+		{
+			var constant = color.GetConstant ();
+			if (constant == null)
+				throw new ArgumentNullException (nameof (color));
+			handle = CGColorGetConstantColor (constant.Handle);
+			if (handle == IntPtr.Zero)
+				throw new ArgumentException (nameof (color));
```

Comment:
I wonder if we should provided a little more of information like `ObjectDisposedException` here I know this one in particular does not apply here but maybe adding something more meaningful in the exception message? Like "Could not create a native instance of {nameof (color)}" or something along those lines.

Reply:
Ideally this type, like others, should move to subclass `NativeObject` and we would let the `Handle` setting code to _handle_ it